app/selectors/index.js

```diff
@@ -18,7 +18,7 @@ const checkDataValid = createSelector(
 );
 
 const checkChartTypeSelected = ({ chartType }) =>
-  chartType.hasOwnProperty('config') && chartType.hasOwnProperty('defaultOpts');
+  Boolean(chartType.config && chartType.defaultOpts);
```

Comment:
See above comment on `hasOwnProperty` in app/actions/index.js. Could also use `!!` here instead of casting explicitly?